### PR TITLE
Kempe's Algorithm in Java Implemented

### PR DESCRIPTION
**Pull request by Riyazul555**

### CODE REVIEW

**Comment by Riyazul555** [comment]
Closes pR #48

File: Kempe's Algorithm/kempeAlgo.java
Comment (iuliagroza) [inline Kempe's Algorithm/kempeAlgo.java:1]: Please rename the file to "Kempe/Kempe.java".

File: Kempe's Algorithm/kempeAlgo.java [hunk 1]
```diff
@@ -0,0 +1,215 @@
+import java.io.*;
```
Comment (iuliagroza) [inline Kempe's Algorithm/kempeAlgo.java:1]: Could you add a header to the file mentioning time and space complexities? You can follow [this example](https://github.com/iuliagroza/Algorithms/blob/57ba92be351961c7909e0b26b263c8bb5c558f69/Merge%20Sort/MergeSort.java#L1), by also denoting what variables you use in the notation.

**Review by iuliagroza — APPROVED** [review]
Super clean implementation! Thanks! Please solve the issues related to naming conventions and the header of the file before merging

File: Kempe's Algorithm/kempeAlgo.java
Comment (Riyazul555) [reply Kempe's Algorithm/kempeAlgo.java:1]: Thanks @iuliagroza ......Sure I will fix the other issues too

File: Kempe's Algorithm/kempeAlgo.java (hunk 1 above)
Comment (Riyazul555) [reply Kempe's Algorithm/kempeAlgo.java:1]: Sure @iuliagroza

**Comment by Riyazul555** [comment]
Done .......As you asked for @iuliagroza

**Comment by Riyazul555** [comment]
Should I merge this PR @iuliagroza

**Pull request merged**